Makefile

```diff
@@ -7,11 +7,22 @@ build:
 	cargo build --release
 
 .PHONY: test
-test:
-	RUST_LOG=$(LOG_LEVEL) cargo test --all --no-fail-fast -- --nocapture --include-ignored
+test: lint test-unit test-integration
+
+.PHONY: lint 
+lint:
 	cargo clippy --all-targets --all-features -- -D warnings
 	cargo fmt --all -- --check
 
+.PHONY: test-unit
+test-unit:
+	RUST_LOG=$(LOG_LEVEL) cargo test --no-fail-fast -- --skip integration_tests --nocapture --include-ignored
+
+.PHONY: test-integration
+test-integration:
```

Comment:
It looks like `make test-integration` is also running unit tests:
```
src/spin [add-deploy-integration-test] $ make test-integration
RUST_LOG=spin=trace cargo test integration_tests::test_dependencies --no-fail-fast -- --nocapture
    Finished test [unoptimized + debuginfo] target(s) in 0.29s
     Running unittests src/lib.rs (target/debug/deps/spin_cli-9890c9e4ab67bd17)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 4 filtered out; finished in 0.00s

     Running unittests src/bin/spin.rs (target/debug/deps/spin-8e3437ebb5e6a5ea)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running tests/integration.rs (target/debug/deps/integration-c72b827f23a47925)

running 1 test
[...]
```

Reply:
Updated `cargo test` command to fix it. Thank you.

```console
$ make test-integration
RUST_LOG=spin=trace cargo test --test integration --no-fail-fast  -- integration_tests::test_dependencies --nocapture
    Finished test [unoptimized + debuginfo] target(s) in 0.33s
     Running tests/integration.rs (target/debug/deps/integration-02d2ab06f0b17d5a)

running 1 test
test integration_tests::test_dependencies ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 10 filtered out; finished in 0.00s

RUST_LOG=spin=trace cargo test --test integration --no-fail-fast -- --skip integration_tests::test_dependencies --nocapture --include-ignored
    Finished test [unoptimized + debuginfo] target(s) in 0.40s
     Running tests/integration.rs (target/debug/deps/integration-02d2ab06f0b17d5a)

running 10 tests
[...]
```